src/memory.rs

```diff
@@ -55,6 +55,9 @@ impl Pointer {
     pub fn points_to_zst(&self) -> bool {
         self.alloc_id == ZST_ALLOC_ID
     }
+    pub fn to_int(&self) -> usize {
+        self.offset
```

Comment:
@oli-obk I don't understand. I'm saying this method should only work for integer pointers, since it treats the pointer as an integer.